app/controllers/custom_gradebook_columns_api_controller.rb

```diff
@@ -26,6 +26,16 @@
 #       "id": "CustomColumn",
 #       "description": "",
 #       "properties": {
+#         "id": {
+#           "description": "The ID of the custom gradebook column",
+#           "example": "2",
```

Comment:
Hi @justin0022 , can you push up a new commit with this example value ("2") unquoted? Since it's an integer, we want to drop the quotes here in the example.